convert_to_multi_instance.sh

```diff
@@ -53,6 +60,205 @@ function prepare_cluster() {
+# function re-install_singleton() {
+#     title "restarting operand-deployment-lifecycle-manager pod"
+#     msg "-----------------------------------------------------------------------"
+
+#     local pod=$(oc get pods -n ${cs_operator_namespace} | grep operand-deployment-lifecycle-manager | awk '{print $1}')
+#     ${OC} delete pod ${pod} -n ${cs_operator_namespace} 
+#     if [[ $? -ne 0 ]]; then
+#         error "Error deleting pod ${pod} in namespace ${cs_operator_namespace}"
+#     fi
+
+# }
+
+#  install new instances of CS based on cs mapping configmap
+function install_new_CS() {
+    title "install new instances of CS based on cs mapping configmap"
+    msg "-----------------------------------------------------------------------"
+
+    ${OC} get configmap common-service-maps -n kube-public -o yaml | while read -r line; do
+        first_element=$(echo $line | awk '{print $1}')
+        
+        if [[ "${first_element}" == "-" ]]; then
+            namespace=$(echo $line | awk '{print $2}')
+            if [[ "${namespace}" != "requested-from-namespace:" ]]; then
+                if [[ "${namespace}" != "${cs_operator_namespace}" ]]; then
+                    return_value=$("${OC}" get namespace ${namespace} || echo failed)
+                    if [[ $return_value != "failed" ]]; then
+                        echo In_CloudpakNS:${namespace}
+                        get_sub=$("${OC}" get sub ibm-common-service-operator -n ${namespace} || echo failed)
+                        if [[ $get_sub == "failed" ]]; then
+                            create_operator_group "${namespace}"
+                            install_common_service_operator_sub "${namespace}"
+                        fi
+                    fi  
+                fi
+            fi
+        fi
+
+        if [[ "${first_element}" == "map-to-common-service-namespace:" ]]; then
+            return_value=$("${OC}" get namespace ${namespace} || echo failed)
+            if [[ $return_value != "failed" ]]; then
+                namespace=$(echo $line | awk '{print $2}')
+                echo In_MasterNS:${namespace}
+                get_sub=$("${OC}" get sub ibm-common-service-operator -n ${namespace} || echo failed)
+                if [[ $get_sub == "failed" ]]; then
+                    create_operator_group "${namespace}"
+                    install_common_service_operator_sub "${namespace}"
+                fi
+            fi  
+        fi
+    done
+    
+    success "Common Services Operator is converted to multi_instance mode"
+}
+
+function create_operator_group() {
+    local CS_NAMESPACE=$1
+
+    title "Creating operator group ..."
+    msg "-----------------------------------------------------------------------"
+
+
+    cat <<EOF | tee >(oc apply -f -) | cat
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: common-service
+  namespace: ${CS_NAMESPACE}
+spec:
+  targetNamespaces:
+  - ${CS_NAMESPACE}
+EOF
+
+    if [[ $? -ne 0 ]]; then
```

Comment:
no need for this if statement. For now just remove it. We will add better error handling later